api/app/Extensions/Socialite/SocialiteManager.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Extensions\Socialite;
+
+use App\Extensions\Socialite\One\TwitterProvider;
+use App\Extensions\Socialite\Two\FacebookProvider;
+use App\Extensions\Socialite\Two\GoogleProvider;
+use Laravel\Socialite\SocialiteManager as Manager;
+use League\OAuth1\Client\Server\Twitter as TwitterServer;
+
+class SocialiteManager extends Manager
+{
+    /**
+     * Create an instance of the specified driver.
+     *
+     * @return \Laravel\Socialite\Two\AbstractProvider
+     */
+    protected function createFacebookDriver()
+    {
+        $config = $this->app['config']['services.facebook'];
```

Comment:
It is preferable to use the Config method for accessing configuration. See http://lumen.laravel.com/docs/installation#accessing-configuration-values